dvc/logger.py

```diff
@@ -186,6 +186,9 @@ def info(msg):
 
     @classmethod
     def box(cls, msg, border_color=''):
+        if cls.logger().level == logging.CRITICAL:
```

Comment:
Let's  move this into a staticmethod Logger.is_quiet() and use it here and down below.

Reply:
But it wouldn't create a circular dependency trying to import `Logger` inside `Progress`?
It would make it more readable, so it's great, but I'm afraid we can't use that one inside `Progress`

Reply:
Hm, good point. Does dynamic import inside Progress._print() work?

Reply:
Let me try it

Reply:
it looks like it worked, @efiop ! I wasn't aware of that wizardy :stuck_out_tongue:

Reply:
Nice! :tada: